examples/run_squad.py

```diff
@@ -855,7 +855,7 @@ def main():
     global_step = 0
     if args.do_train:
         cached_train_features_file = args.train_file+'_{0}_{1}_{2}_{3}'.format(
-            args.bert_model, str(args.max_seq_length), str(args.doc_stride), str(args.max_query_length))
+            list(filter(None, args.bert_model.split('/'))).pop(), str(args.max_seq_length), str(args.doc_stride), str(args.max_query_length))
```

Comment:
Could you explain what `filter(None, args.bert_model.split('/'))` please ?

Reply:
It will take care of the case when the path ends with a `/`

Reply:
@thomwolf That's right. It handles without error even if the path ends with `/`. cc @rodgzilla